client/post-editor/editor-ground-control/style.scss

```diff
@@ -156,9 +156,9 @@
 	display: none;
 }
 
+.editor-ground-control__history-button.button.is-link,
```

Comment:
Is the `.button.is-link` for specificity purposes? Otherwise we could kill this off 🔪 

Another option might be to style all `.editor-ground-control .button.is-link`s the same - not sure if this would be a good idea long term though.

Reply:
Same thought process here. Initially, I actually combined the two with a selector that just made all buttons look the same but then reverted back to this as I wasn't sure about other buttons (potentially being added in the future). Then again combining the selector would've probably been the right choice.